prompt/prompt-executor/prompt-executor-clients/prompt-executor-google-client/src/commonMain/kotlin/ai/koog/prompt/executor/clients/google/GoogleLLMClient.kt

```
@@ -192,7 +192,7 @@ public open class GoogleLLMClient(
                         outputTokensCount = it.candidatesTokenCount,
                     )
                 }
-                response.candidates.firstOrNull()?.let { candidate ->
+                response.candidates?.firstOrNull()?.let { candidate ->
```

If `candidates` is now nullable, I would check if it's null first and throw if it is:
```
val candidates = response.candidates ?: throw LLMClientException(clientName, "Null candidates in Google API response")
```
While `firstOrNull()` is fine in this case, there is also `isNotEmpty()` and `isEmpty()` later on on same variable, which is inconsistent, unclear why, and doesn't prevent silent failures.

---

I dived into docs and realized that actually this field shouldn't be nullable. Let me fix it quickly.